src/AppBundle/Controller/DefaultController.php

```diff
@@ -95,6 +96,58 @@ public function newPostAction(Request $request)
     }
     
     /**
+     * @Route("/downvote", name="new_downvote")
+     * @Method({"POST"})
+     */
+    public function newDownvoteAction(Request $request) 
+    {
+        // Get the post_id
+        $post_id = $request->get("$post_id");
+        
+        // Get the post from the post_id in the database
+        $post = $this->getDoctrine()
+                     ->getRepository('AppBundle:Post')
+                     ->find($post_id);
+        
+        // If anything other than a post is returned (including null)
+        // throw an error.
+        if (!$post) {
+            throw $this->createNotFoundException(
+                'No post found for id ' . $id
+            );
+        }
+
+        // We have everything we need now
+        // Time to add the post to the database
+        try {
+            $em = self::getEntityManager();
+            $dislike = new PostDislike;
```

Comment:
Entity is PostLikes, not PostDislike. 

It's inside `src/AppBundle/Entity`. You can see the possible tables there. :)